src/dash-table/derived/filter/map.ts

```diff
@@ -74,36 +74,42 @@ function updateMap(map: Map<string, SingleColumnSyntaxTree>, column: IColumn, va
     return newMap;
 }
 
-function updateState(map: Map<string, SingleColumnSyntaxTree>, setFilter: SetFilter) {
+function updateState(
+    map: Map<string, SingleColumnSyntaxTree>,
+    operator: FilterLogicalOperator,
+    setFilter: SetFilter
+) {
     const asts = Array.from(map.values());
-    const globalFilter = getMultiColumnQueryString(asts);
+    const globalFilter = getMultiColumnQueryString(asts, operator);
 
     const rawGlobalFilter = R.map(
         ast => ast.query || '',
         R.filter<SingleColumnSyntaxTree>(ast => Boolean(ast), asts)
-    ).join(' && ');
+    ).join(` ${operator === FilterLogicalOperator.And ? '&&' : '||'} `);
```

Comment:
```suggestion
    ).join(operator === FilterLogicalOperator.And ? ' && ' : ' || ');
```
not a big deal, but the string interpolation feels less readable to me in this case - as well as a few chars longer 😅

Reply:
👍